examples/inference/api_server_openai/query_http_requests.py

```diff
@@ -14,26 +14,33 @@
 # limitations under the License.
 #
 
-import os
 import json
 import requests
+import argparse
 
-s = requests.Session()
+parser = argparse.ArgumentParser(description="Example script to query with http requests", add_help=True)
+parser.add_argument("--request_api_base", default="http://localhost:8000/v1", type=str, help="Deployed model endpoint url")
+parser.add_argument("--request_model", default="gpt2", type=str, help="The name of model to request")
+parser.add_argument("--streaming_response", default=False, action="store_true", help="Whether to enable streaming response")
+parser.add_argument("--max_new_tokens", default=None, help="The maximum numbers of tokens to generate")
+parser.add_argument("--temperature", default=None, help="The value used to modulate the next token probabilities")
+parser.add_argument("--top_p", default=None, help="If set to float < 1, only the smallest set of most probable tokens with probabilities that add up to`Top p` or higher are kept for generation")
+
```

Comment:
top_k is not supported?

Reply:
Yes, `top_k`  is not included in [openai api](https://platform.openai.com/docs/api-reference/chat/create).